CreeDictionary/utils/paradigm_filler.py

```diff
@@ -118,3 +119,50 @@ def fill_paradigm(
             row[col_ind] = " / ".join(results_for_cell)
 
         return tables
+
+    def inflect_all_with_analyses(
+        self, lemma: str, wordclass: WordClass
+    ) -> Dict[ConcatAnalysis, Sequence[str]]:
+        """
+        Produces all known forms of a given word. Returns a mapping of analyses to their
+        forms. Some analyses may have multiple forms. Some analyses may not generate a
+        form.
+        """
+        analyses = self.expand_analyses(lemma, wordclass)
+        return self._generator.feed_in_bulk_fast(analyses)
+
+    def inflect_all(self, lemma: str, wordclass: WordClass) -> Set[str]:
+        """
+        Return a set of all inflections for a particular lemma.
+        """
+        all_inflections = self.inflect_all_with_analyses(lemma, wordclass).values()
+        return set(form for word in all_inflections for form in word)
```

Comment:
Honestly, I'm not sure if these methods in and of themselves are useful, BUT, they certainly are easier and clearer to test than `expand_analyses()` which they make use of. In particular, `inflect_all()` is straightforward to test, and makes use of `expand_analyses()` (albeit transitively).